### DISCUSSION TITLE
Are there changes in 0.9.3-preview3.1 with register in Startup?

### DISCUSSION THREAD

**Discussion by Riedi2070** [Megabit/Blazorise#1593] [Q&A] [answered]
In the 0.9.3-preview3.1 the compiler failed when adding these lines.

![image](https://user-images.githubusercontent.com/5427990/101674222-83efdc80-3a58-11eb-8fcb-c8bbb06b8798.png)
Are these lines no longer necessary?

**Comment by stsrki** [comment, marked as answer]
Yes, the `.Use` methods are removed and they are not needed anymore. This is because of the newly done optimizations in 0.9.3 and .Net 5 where `IComponentActivator` is used to initialize components. As a result, now the Blazorise is a lot cleaner and a lot faster in runtime.

**Reply by Riedi2070** [reply]
Thx for good job!
